src/index.js

```diff
@@ -113,6 +113,17 @@ class Service {
     return removeBulk(this, params)
       .catch(errorHandler);
   }
+
+  raw (query) {
+    return raw(this, query)
+      .catch(errorHandler);
+  }
+}
+
+function raw(service, query) {
+  const method = query.method;
+  const params = query.query;
```

Comment:
And here we could simply say `const { method, query } = params;` and use `query` below.